src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java

```diff
@@ -109,6 +109,31 @@
         assertFalse("resource is not allowed", p.allowsFrameFromSource(URI.parse("http://www.def.am:555")));
         assertFalse("resource is not allowed", p.allowsChildFromSource(URI.parse("https://www.def.am:555")));
         assertTrue("resource is allowed", p.allowsChildFromSource(URI.parse("http://www.def.am:555")));
+        
+        p = Parser.parse("font-src https://font.com http://font.org", URI.parse("https://abc.com"));
+        assertTrue("resource is allowed", p.allowsFontFromSource(URI.parse("https://font.com")));
+        assertFalse("resource is not allowed", p.allowsFontFromSource(URI.parse("https://font.com:555")));
```

Comment:
Per https://w3c.github.io/webappsec-csp/#match-url-to-source-expression 2.1.1, `font-src https://font.com http://font.org` is equivalent to `font-src https://font.com http://font.org https://font.org` and should match https://font.com, http://font.org, https://font.org. Note that it is not any port, but default port for given scheme: https://url.spec.whatwg.org/#default-port .
So yeah, should assert true, and https://github.com/shapesecurity/salvation/pull/152#discussion-diff-82918521R115 should assert false.